python/shipPatRec.py

```diff
@@ -911,7 +911,6 @@ def tracks_combination_using_extrapolation(recognized_tracks_12, recognized_trac
             atrack['hits_y34'] = recognized_tracks_34[i_34]['hits_y']
             atrack['hits_stereo34'] = recognized_tracks_34[i_34]['hits_stereo']
             recognized_tracks_combo.append(atrack)
-            recognized_tracks_combo.append(atrack)
```

Comment:
Please double-check where this is used, to ensure that nothing depends on entries being duplicated (I hope not, but better to make sure)